### DISCUSSION TITLE
Getting Started?

### DISCUSSION THREAD

**Discussion by jkihgit** [microsoft/pyright#2599] [Q&A]
I've looked at the README.md and installed according to the directions
Specifically, using Sublime Text's Package Control to install LSP-pyright, then restart Sublime Text
The README then told me to install node js is prompted to (I wasnt)

But I dont see what I should do after this

I tried using the default typing interface:
```python
class MyObject():
    def __init__(self, name:str=""):
        self.name = name
```
However, I dont get any errors when I run this in Sublime Text using Ctrl+B with
```python
myobject = MyObject(0)
```

Is there further resource I could reference?

**Comment by erictraut** [comment]
You are probably better off posting this question in the [LSP-pyright](https://github.com/sublimelsp/LSP-pyright) repo.
